docs/source/openlane_commands.md

```diff
@@ -18,12 +18,12 @@ Most of the following commands' implementation exists in this [file][0]
 | Command      | Flags                   | Description                                           |
 |---------------|------------------------|-----------------------------------------|
 | `set_netlist <netlist>`   | | Sets the current netlist used by the flow to `<netlist>` |
-|    | `[-lec]` | Runs logic verification for the new netlist against the previous netlist. <br> Optional flag.       |
+|    | `[-lec]` | Runs logic verification for the new netlist against the previous netlist, if `LEC_ENABLE` is set to 1. <br> Optional flag. |
```

Comment:
isn't the whole purpose of using `-lec` is to force it despite the env variable?